# Normal Curve Calculations and the $z$-Proportion Test

Let's begin with some normal distrubutions so that we can work with the properties of standardized scores and percentiles in these distributions.

The SAT has the distribution:

$$N(1000, 200)$$

while the ACT has the distribution:

$$N(21,5)$$

The classic IQ distribution has the distribution:

$$N(100,15)$$

## Z-Scores and Percentiles

For every z-score we calculate, we can determine the precise percentile. For example, the percentile corresponding to a $z$-score of $z=2$ is given by:

In [3]:
pnorm(2)

[1] 0.9772499

Hence, the 97th percentile corresponds to standardized score of 2.

### Example 1

Calculate the $z$-score and percentile for someone who scored a 1270 on the SAT.

Note that we use the function **pnorm()** to determine the perecentile that correponds to a $z$-score.

In [9]:
z <- (1270-1000)/200
z
pnorm(z)

[1] 1.35

[1] 0.911492

Hence, for a SAT score of 1270:
- The standardized score is 1.35, and
- The correponding percentile is 91st.

## Example 1b

What is the 95th percentile score on the SAT?

We use the function **qnorm()** to convert percentiles back into $z$-scores:

In [10]:
z = qnorm(0.95)
z
SAT = 1000 + 200 * z

[1] 1.644854

## Example 2

Sally earned a 31 on the ACT while Joanna earned a 1380 on the SAT. Who did better relative to the test she took?

In [12]:
zs = (31 - 21) / 5
zs
pnorm(zs)

[1] 2

[1] 0.9772499

In [13]:
zj = (1380 - 1000) / 200
zj
pnorm(zj)

[1] 1.9

[1] 0.9712834

The $z$-score for Sally was $2.0$ compared a $z$-score of $1.9$ for Joanna. Thus, with a larger positive $z$-score, we know that Sally was further above average in units of standard deviations than was Joanna.

The percentiles are not very different at all, yet Sally's exactly percentile is approximately $0.97725$ compared to Joanna's which was approximately $0.97128$.

## Example 3

What is the 80th percentile IQ score?

Let's use the variable **z8** to refer to the standardized score that corresponds to the 80th percentile.

In [15]:
z8 = qnorm(0.8)
z8

[1] 0.8416212

In [16]:
IQ = 100 + 15 * 0.84
IQ

[1] 112.6

Thus, an IQ of about 112 is the 80th percentile for the IQ distribution.

## Dolphins and the Proportion Test in R

The classic case study from Rossman and Chance tells the story of a group of patients with anxiety who travel to the Carribean for time at a resort where they also received group therapy sessions. The treatment group, along with all the resort living and group therapy, also have a chance to swim with dolphins.

In [25]:
dolphin <- read.csv('http://faculty.ung.edu/rsinn/data/dolphin.csv')
dolphin

Treatment,Result
<chr>,<chr>
Dolphins,Improved
Dolphins,Improved
Dolphins,Improved
Dolphins,Improved
Dolphins,Improved
Dolphins,Improved
Dolphins,Improved
Dolphins,Improved
Dolphins,Improved


Let's summarize the results with the **xtabs()** function:

In [27]:
xtabs(~dolphin$Treatment + ~dolphin$Result)

                 dolphin$Result
dolphin$Treatment Did Not Improved
         Control       12        3
         Dolphins       5       10

From the above 2-way table, we can gather that we have:

- 10 successes and 3 failures in the treatment group.
- 3 successes and 12 failures in the control group.

We need to create two vectors. The first vector we create will include the successes. The second will include the totoals per group.

In [30]:
# Number of successes

x <- c(3,10)

# Total participants

n <- c(15,15)

In [31]:
ztest <- prop.test(x, n, alternative = 'less')
ztest


	2-sample test for equality of proportions with continuity correction

data:  x out of n
X-squared = 4.8869, df = 1, p-value = 0.01353
alternative hypothesis: less
95 percent confidence interval:
 -1.0000000 -0.1374333
sample estimates:
   prop 1    prop 2 
0.2000000 0.6666667 


By default, the **prop.test()** function conducts a $\chi^2$ test instead of a $z$-test for population proportion. However, the $z$ has been run, and we can find the test statistic from it as follows:

In [32]:
ztest$statistic

X-squared 
 4.886878

The $z$-statistic in this case is the square root of the $\chi^2$ statistic:

In [33]:
z = sqrt(4.8868778280543)
z

[1] 2.210628

## Example 4

The smoking rate in a town historically has been 21%. Has the rate changed given that, in 2025, a survey of 100 randomly selected residents of the town found that 14 of them were smokers.